spec/lib/reports/brexit_subscribers_report_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Reports::BrexitSubscribersReport do
+  describe "#call" do
+    let!(:date) { Time.zone.local(2020, 7, 12) }
+    let(:list) { create(:subscriber_list, :brexit, title: "brexit", slug: "/brexit") }
+    let(:subscriber) { create(:subscriber) }
+    let!(:subscription) do
+      create(:subscription, :daily,
+             subscriber: subscriber,
+             subscriber_list: list,
+             created_at: (date + 1.day))
+    end
+
+    context "date is given" do
+      it "returns all brexit lists subscribed to before a date" do
```

Comment:
This it condition doesn't seem to quite match with what is being tested - since the test is checking that an empty output.

I think you might want to have 3 scenarios tested with 2 subscriptions existing in the system: it returns all subscriptions when not given a date (returns the list with 2 results), it returns all subscriptions created before a date (returns the list with 1 result), and it returns an empty CSV when no subscriptions were created before the date. 

What do you think?

Reply:
Sounds good - thanks!